guides/common/modules/con_comparing-multiple-content-view-environments-and-composite-content-views.adoc

```diff
@@ -0,0 +1,20 @@
+[id="comparing_multiple_content_view_environments_and_composite_content_views_{context}"]
+= Comparing multiple content view environments and composite content views
+
+Composite content views provide an alternative method for granting hosts access to content from multiple content views.
+You can use composite content views, multiple content view environments, or a combination of both.
+The key differences between these methods include the following:
+
+* Access Timing:
```

Comment:
These don't show up as bold in the rendered doc. Do they need trailing asterisks?

Reply:
They did need that. Fixing it.